client/my-sites/customer-home/cards/education/mastering-gutenberg/index.jsx

```diff
@@ -0,0 +1,122 @@
+ */
+import './style.scss';
+
+class MasteringGutenberg extends Component {
+	state = {
+		activeDialog: false,
+	};
+
+	handleDialogClosure = () => {
+		this.setState( {
+			showDialog: false,
+			activeDialog: false,
+		} );
+	};
+
+	handleButtonClick = button => {
+		this.setState( { showDialog: true, activeDialog: button } );
+	};
+
+	render() {
+		const { cannotUseGutenberg, translate } = this.props;
+		const { activeDialog, showDialog } = this.state;
+
+		if ( cannotUseGutenberg ) {
+			return null;
+		}
+
+		let supportLink;
+		let videoId;
+
+		switch ( activeDialog ) {
+			case 'adjustingBlocks':
+				videoId = 'oQNap9YTrWw';
+				supportLink = localizeUrl( 'https://wordpress.com/support/wordpress-editor' );
+				break;
+			case 'customizingWithBlocks':
+				videoId = 'BkQlTuJ4BH8';
+				supportLink = localizeUrl( 'https://wordpress.com/support/wordpress-editor/blocks/' );
+				break;
+		}
+
+		return (
+			<Fragment>
+				<Dialog additionalClassNames="mastering-gutenberg__dialog" isVisible={ showDialog }>
+					<iframe
+						title={ translate( 'Video demonstration' ) }
+						width="560"
+						height="315"
+						src={ `https://www.youtube.com/embed/${ videoId }?rel=0&autoplay=1` }
+						frameBorder="0"
+						allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+						allowFullScreen={ true }
+					/>
+					<div className="mastering-gutenberg__dialog-buttons">
+						<Button onClick={ this.handleDialogClosure }>{ translate( 'Close' ) }</Button>
+						<Button href={ supportLink } primary target="_blank">
+							{ translate( 'Learn more' ) }
+							<Gridicon icon="external" />
+						</Button>
+					</div>
+				</Dialog>
+				<DismissibleCard
+					preferenceName="customer-home-mastering-gutenberg-card"
+					className="mastering-gutenberg"
+				>
+					{ ! isMobile() && (
+						<div className="mastering-gutenberg__illustration">
+							<img src="/calypso/images/illustrations/gutenberg-mini.svg" alt="" />
+						</div>
+					) }
+					<div>
+						<CardHeading>{ translate( 'Master the Block Editor' ) }</CardHeading>
+						<p className="mastering-gutenberg__text customer-home__card-subheader">
+							{ translate(
+								'Learn how to create stunning post and page layouts ' + 'through our video guides.'
+							) }
+						</p>
+						<Button
+							onClick={ this.handleButtonClick.bind( this, 'adjustingBlocks' ) }
+							className="mastering-gutenberg__link is-link"
+						>
+							{ translate( 'Adjusting settings of blocks' ) }
+						</Button>
+						<Button
+							onClick={ this.handleButtonClick.bind( this, 'customizingWithBlocks' ) }
+							className="mastering-gutenberg__link is-link"
+						>
+							{ translate( 'Customizing posts and pages with blocks' ) }
+						</Button>
+					</div>
+				</DismissibleCard>
+			</Fragment>
+		);
+	}
+}
+
+export default connect( state => {
+	return {
+		cannotUseGutenberg: isClassicEditorForced( state, getSelectedSiteId( state ) ),
```

Comment:
Good thinking here! 👍 We'll probably move this check over to the layout API but this is a great reminder ✨

Reply:
Should I remove it here now? Just checking as it seems other cards have display logic within them too. :)

https://github.com/Automattic/wp-calypso/blob/1d38c53dba02a5930b39fac2ab3c23f7ba52d6a1/client/my-sites/customer-home/cards/features/stats/index.jsx#L37-L39

https://github.com/Automattic/wp-calypso/blob/1d38c53dba02a5930b39fac2ab3c23f7ba52d6a1/client/my-sites/customer-home/cards/features/grow-earn/index.jsx#L32-L34

Reply:
We can follow up later once they are moved to the API, no need to remove them for now.

Note that `isClassicEditorForced` will return false when the selected editor is the classic one. So I wonder if we should show the card only when the selected editor is Gutenberg: `cannotUseGutenberg: getSelectedEditor( state, siteId ) === 'classic'`;

Reply:
> We can follow up later once they are moved to the API, no need to remove them for now.

Actually, since the card won't be displayed until we update the API, let's remove the checks now. We'll make sure the card won't be returned by the API is Gutenberg is not the selected editor.

Reply:
> So I wonder if we should show the card only when the selected editor is Gutenberg: cannotUseGutenberg: getSelectedEditor( state, siteId ) === 'classic'

There was some discussion about this [here](https://github.com/Automattic/wp-calypso/pull/39617#issuecomment-591086780):

> While I agree that it would make sense to hide the card if a person explicitly doesn't want to use Gutenberg, the community keeps adding new functionality and fixing things that didn't work well in a previous release. A Dismiss option works best for serving all of those intentions.

Now that it's just an ordinary card and not a dismissable one, I'm not sure that still applies. 

I've removed the checks anyway. :)